diff --git a/asyn/drvAsynSerial/drvAsynSerialPortWin32.c b/asyn/drvAsynSerial/drvAsynSerialPortWin32.c
index 20fe15b8..6aa2b867 100644
--- a/asyn/drvAsynSerial/drvAsynSerialPortWin32.c
+++ b/asyn/drvAsynSerial/drvAsynSerialPortWin32.c
@@ -479,7 +479,7 @@ static asynStatus writeIt(void *drvPvt, asynUser *pasynUser,
     const char *data, size_t numchars,size_t *nbytesTransfered)
 {
     ttyController_t *tty = (ttyController_t *)drvPvt;
-    int thisWrite;
+    DWORD thisWrite;
     int nleft = (int)numchars;
     int timerStarted = 0;
     BOOL ret;
@@ -552,7 +552,7 @@ static asynStatus readIt(void *drvPvt, asynUser *pasynUser,
     char *data, size_t maxchars,size_t *nbytesTransfered,int *gotEom)
 {
     ttyController_t *tty = (ttyController_t *)drvPvt;
-    int thisRead;
+    DWORD thisRead;
     int nRead = 0;
     int timerStarted = 0;
     COMMTIMEOUTS ctimeout;
